packages/sdk-v2/src/sdk/mpbridge/types.ts

```diff
@@ -0,0 +1,128 @@
+import { ethers } from "ethers";
+import { TransactionStatus } from "@usedapp/core";
+import bridgeContracts from "@gooddollar/bridge-contracts/release/mpb.json";
+import { SupportedChains } from "../constants";
+
+export enum BridgeService {
```

Comment:
This should be inverted. axelar = 0, layer-zero=1
as read from https://docs.gooddollar.org/user-guides/bridge-gooddollars